arch/arm/configs/tsimx6ul_minimal_defconfig

```diff
@@ -78,16 +78,16 @@ CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
 CONFIG_IMX_WEIM=y
 CONFIG_TSPC104=y
 CONFIG_CONNECTOR=y
-CONFIG_MTD=m
+CONFIG_MTD=y
 # CONFIG_MTD_OF_PARTS is not set
-CONFIG_MTD_BLOCK=m
+CONFIG_MTD_BLOCK=y
 CONFIG_MTD_CFI=m
 CONFIG_MTD_JEDECPROBE=m
 CONFIG_MTD_CFI_INTELEXT=m
 CONFIG_MTD_CFI_AMDSTD=m
 CONFIG_MTD_CFI_STAA=m
 CONFIG_MTD_SST25L=m
-CONFIG_MTD_SPI_NOR=m
+CONFIG_MTD_SPI_NOR=y
```

Comment:
Is this actually necessary to have as a builtin? IIRC, these should load automatically if they are correctly defined in the FDT. Was there a situation where the modules weren't loading?

I'm operating and questioning this on the assumption that, things that are critical to operation of a device or its boot speed, should be builtin; everything else can usually be okay being a module. Since MTD isn't needed for boot, nor early operation of the device, I'd think so long as these are being correctly loaded they could stay as modules.